projects/mars-theme/frontity.settings.js

```diff
@@ -8,6 +8,7 @@ const settings = {
     },
   },
   packages: [
+    "@frontity/amp",
```

Comment:
Hmm, do you want to add the AMP package for some specific reason to mars-theme or was it just something you did for testing?

Reply:
Gaah! Totally forgot to remove them sorry! On it right now!

Reply:
```suggestion
```
Remove it!